packages/sdk/src/oracle/services/optimisticOracle.ts

```diff
@@ -0,0 +1,22 @@
+import { OptimisticOracleEthers, OptimisticOracleEthers__factory } from "@uma/contracts-node";
+import { BigNumberish, BigNumber, Provider } from "../types/ethers";
+
+type Props = {
+  liveness: BigNumber;
+}
+type Request = ReturnType<OptimisticOracleEthers['getRequest']>
+
+export class OptimisticOracle {
+  public contract: OptimisticOracleEthers;
+  constructor(protected provider: Provider, protected address: string) {
+    this.contract = OptimisticOracleEthers__factory.connect(address, provider);
+  }
+  async getRequest(requester:string,identifier:string, timestamp:BigNumberish, ancillaryData:string): Promise<Request>{
+    return this.contract.callStatic.getRequest(requester, identifier, timestamp, ancillaryData)
+  }
+  async getProps():Promise<Props>{
+    return {
+      liveness: await this.contract.defaultLiveness()
```

Comment:
Do you think the name "liveness" is a bit misleading since some requests may specify a different liveness? Or do you think this self-explanatory since the request object should have a liveness value as well.

Reply:
yea you are right, i should change this to be more literal, "defaultLiveness"

Reply:
im not 100% sure, but it seems if a request does not override the liveness, liveness wont available on the request object which makes this call necessary.  is that the case?